tests/integration/profile/test_podman_compose_config.py

```diff
@@ -80,3 +80,22 @@ def test_config_profiles(self, profiles, expected_services):
             actual_services[service] = service in actual_output
 
         self.assertEqual(expected_services, actual_services)
+
+    def test_config_quiet(self):
+        """
+        Tests podman-compose config command with the --quiet flag.
+        """
+        config_cmd = [
+            "coverage",
+            "run",
+            podman_compose_path(),
+            "-f",
+            profile_compose_file(),
+            "config",
+            "--quiet",
+        ]
+
+        out, _ = self.run_subprocess_assert_returncode(config_cmd)
+
+        string_output = out.decode("utf-8")
+        self.assertEqual("", string_output)
```

Comment:
I think `string_output` is not needed and can be inlined - code is obvious anyway. Also, assertEqual(actual, expected) is more readable in my opinion, though yes, this is not followed consistently in the codebase.